integration-tests/startup/index.js

```diff
@@ -10,12 +10,26 @@ if (process.env.AGENT_URL) {
   options.url = process.env.AGENT_URL
 }
 
+if (process.env.SCOPE) {
+  options.scope = process.env.SCOPE
+}
+
 require('../..').init(options)
 
 const http = require('http')
 
 const server = http.createServer((req, res) => {
-  res.end('hello, world\n')
+  process.nextTick(() => {
+    setImmediate(() => {
+      (async () => {
+        await new Promise(resolve => {
+          resolve()
+        }).then(() => {
+          res.end('hello, world\n')
```

Comment:
The plugin attaches the span to the request object and doesn't use the scope manager, so this doesn't actually end up testing it.

Reply:
Would adding a tag to the active span suffice here then?

Reply:
What is this actually testing? If it's testing the scope manager maybe it should just be used directly.